index.html

```diff
@@ -6,8 +6,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
     <title>Portfolio Skeleton</title>
+    <script src="https://kit.fontawesome.com/5211805414.js" crossorigin="anonymous"></script>
 </head>
 <body>
```

Comment:
- [ ] Please I highly recommend sticking to the [FIGMA DESIGN TEMPLATE](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=wZKMjYgOqOhSRgOL-0). 😄 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (232)](https://user-images.githubusercontent.com/78104544/234750172-015ae582-8aab-44f2-b5b9-73ae5af7747b.png)


_**YOUR DESIGN**_

- [ ] Please I kindly suggest using the `h1` "HEY THERE" text instead of "Greetings". 😄 
- [ ] Please I suggest adding `margin` to the icons to move to the left. 😄 
- [ ] Please I suggest changing the `h3` text from `I'll become a software developer` to `I'm a software developer`. 😄 
- [ ] Please I highly recommend using a specific logo design on the top left corner instead of the Microverse logo since this project is a personal portfolio website. 😄 

![Screenshot (233)](https://user-images.githubusercontent.com/78104544/234750234-cbb9f2e8-0ae2-4a04-9f09-0cb7b331945e.png)